config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -4940,6 +4940,11 @@
 # Default : /portal/help/TOCDisplay/content.hlp?docId=howdoiaddlatexlanguagetomycoursesite
 #portal.mathjax.help.url=https://sakai.screenstepslive.com/s/sakai_help/m/68426/l/728838-how-do-i-add-latex-language-to-my-course-site
 
+# Select one or more formats for MathJax. The default one being LaTeX.
+#mathjax.config.format.count=2
```

Comment:
shall we talk about extensions?

mathjax.extension.count ?

Reply:
_MathJax.Hub.Config ({
    "extensions":["tex2jax.js","asciimath2jax.js"],
    "jax":["input/TeX","input/AsciiMath","output/HTML-CSS"],
    "messageStyle":"none",
    "tex2jax":{"inlineMath":[["$$","$$"],["\\(","\\)"]]
}});_

Based on the properties, I modify the extensions but also input (jax section), that's why I finally didn't use the word "extension".

But sure, there might be a better name. Maybe just `mathjax.format.count`.

Reply:
Nah, it's fine, mathjax.config.format is enough.

Reply:
Put the JIRA in the comment, "SAK-47442 - Mathjax can use multiple formats, enable your instance supported formats with these properties."